docs/packages/pkg/libmill.rst

```diff
@@ -0,0 +1,22 @@
+.. spelling::
+
+    libmill
+
+.. index:: unsorted ; libmill
+
+.. _pkg.libmill:
+
+libmill
+=======
+
+-  `Official <https://github.com/sustrik/libmill>`__
+-  `Hunterized <https://github.com/hunter-packages/libmill>`__
+-  `Example <https://github.com/ruslo/hunter/blob/master/examples/libmill/CMakeLists.txt>`__
+-  Added by `Isaac Hier <https://github.com/isaachier>`__ (`pr-N <https://github.com/ruslo/hunter/pull/N>`__)
```

Comment:
Pull request number:
* https://docs.hunter.sh/en/latest/creating-new/create/cmake.html#pull-requests

Reply:
Sure, I also need you to fork the repo please.

Reply:
Done: https://github.com/hunter-packages/libmill